### DISCUSSION TITLE
Request for Removal of Unauthorized Code

### DISCUSSION THREAD

**Discussion by GauriLogitax** [community/community#118508] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Dear GitHub Team,

I am writing to you on behalf of Logitax Pvt. Ltd. regarding an issue with unauthorized code usage on your platform. Our company has discovered that proprietary code belonging to Logitax Pvt. Ltd. has been posted without authorization on GitHub.

Company Name: Logitax.
Company URL: https://logitax.in/

The unauthorized code is hosted in the following repository:
Repository URL: https://github.com/harindran/Common-EInvoicingLogitaxAPI

We kindly request that you take immediate action to remove this code from your platform as it violates our intellectual property rights. Logitax Pvt. Ltd. has not granted permission for the distribution or use of this code, and its presence on GitHub is unauthorized.

We appreciate your prompt attention to this matter and request that you provide confirmation once the unauthorized code has been removed.

If you require any further information or documentation from our end to facilitate the resolution of this issue, please do not hesitate to contact us.

Thank you for your cooperation.

**Comment by jfullstackdev** [comment]
I'm not a GitHub Staff, just want to ask how they have the code stolen from your company ?

**Comment by ebndev** [comment]
Hi [GauriLogitax](https://github.com/GauriLogitax),

The best place to go for answers to account related questions can be found on our [Support page](https://support.github.com/contact) and you can open a ticket there if needed.

The Community Discussions team is unable to assist with nor escalate your issue so I am going to close this discussion.

Thank you for understanding.
